site-src/guides/use-cases.md

```diff
@@ -0,0 +1,130 @@
+# Common Use Cases
+
+??? example "Experimental"
+
+    This project is still in an alpha state and breaking changes may occur in the future.
+
+This section provides examples to address various generative AI application scenarios by using Gateway API Inference Extension.
+
+## Example 1: Serve multiple generative AI models
+A company wants to deploy multiple large language models (LLMs) to serve different workloads. 
+For example, they might want to deploy a Gemma3 model for a chatbot interface and a Deepseek model for a recommendation application. 
+The company needs to ensure optimal serving performance for these LLMs.
+Using Gateway API Inference Extension, you can deploy these LLMs on your cluster with your chosen accelerator configuration in an `InferencePool`. 
+You can then route requests based on the model name (such as chatbot and recommender) and the `Criticality` property.
+
+The following diagram illustrates how Gateway API Inference Extension routes requests to different models based on the model name.
+![Serving multiple generative AI models](../images/serve-mul-gen-AI-models.png)
+
+This example illustrates a conceptual example regarding how you can use the `HTTPRoute` object to route based on model name like “chatbot” or “recommender” to `InferencePool`.
+```shell
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: routes-to-llms
+spec:
+  parentRefs:
+  - name: inference-gateway
+  rules:
+  - matches:
+    - headers:
+      - type: Exact
+        name: X-Gateway-Model-Name
```

Comment:
Additional context should be provided, e.g. [BBR](https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/main/pkg/bbr/README.md) is being used to copy the model name from the request body to the header.

Reply:
done